Content.Server/GameObjects/Components/PlaceableSurfaceComponent.cs

```diff
@@ -31,6 +34,39 @@ public override bool IsPlaceable
             }
         }
 
+        public override bool PlaceCentered
+        {
+            get => _placeCentered;
+            set
+            {
+                if (_placeCentered == value)
+                {
+                    return;
+                }
+
+                _placeCentered = value;
+
+                Dirty();
```

Comment:
No point calling dirty on these when they're not on the state that gets sent from GetComponentState (probably best to add them to the state and then also handle it client-side).

Reply:
I've removed the call from the setter and added the variables to the state. Should I do the same for the IsPlaceable variable? since it's already in the ComponentState and has get/set on the client-side too

Reply:
Now you have the opposite problem! These variables are on the state, but you're not calling `Dirty()` after changing them...

Reply:
Soo I need to call Dirty() like before? (Now the vars are in the state)

Reply:
Yes!

Reply:
Thanks for the feedback! Made the changes.